ninja_extra/pagination/models/page_by_number.py

```diff
@@ -48,7 +48,7 @@ def __init__(
     def create_input(self) -> Type[Input]:
         class DynamicInput(PageNumberPaginationExtra.Input):
             page: int = Field(1, gt=0)
-            page_size: int = Field(self.page_size, lt=self.max_page_size)
+            page_size: int = Field(self.page_size, gt=0, lt=self.max_page_size)
```

Comment:
And this `Field(100, gt=0, le=self.max_page_size, lt=self.max_page_size + 1)`

Reply:
Same question as above, why both `lt` and `le` ?

Reply:
`lt=self.max_page_size + 1` like you suggested